test/sharness/t0087-repo-robust-gc.sh

```diff
@@ -155,12 +157,12 @@ test_gc_robust_part2() {
 test_init_ipfs
 
 test_gc_robust_part1
-test_gc_robust_part2
+#test_gc_robust_part2
 
 test_launch_ipfs_daemon_without_network
 
 test_gc_robust_part1
-test_gc_robust_part2
+#test_gc_robust_part2
```

Comment:
Why are these disabled?

Reply:
I have no memory of having to disable any tests. You should re-enable.